contracts/modules/VersionManager.sol

```diff
@@ -234,6 +239,41 @@ contract VersionManager is IVersionManager, IModule, BaseFeature, Owned {
         guardianStorage.revokeGuardian(_wallet, _guardian);
     }
 
+    /**
+     * @inheritdoc IVersionManager
+     */
+    function setLock(address _wallet, uint256 _releaseAfter) external override onlyFeature(_wallet) {
+        lockStorage.setLock(_wallet, msg.sender, _releaseAfter);
+    }
+
+    /**
+     * @inheritdoc IVersionManager
+     */
+    function setLimit(address _wallet, ILimitStorage.Limit memory _limit) external override onlyFeature(_wallet) {
+        limitStorage.setLimit(_wallet, _limit);
+    }
+
+    /**
+     * @inheritdoc IVersionManager
+     */
+    function setDailySpent(address _wallet, ILimitStorage.DailySpent memory _dailySpent) external override onlyFeature(_wallet) {
+        limitStorage.setDailySpent(_wallet, _dailySpent);
+    }
+
+    /**
+     * @inheritdoc IVersionManager
+     */
+    function setLimitAndDailySpent(
+        address _wallet,
+        ILimitStorage.Limit memory _limit,
+        ILimitStorage.DailySpent memory _dailySpent
+    ) 
+        external
+        override
+        onlyFeature(_wallet) 
+    {
+        limitStorage.setLimitAndDailySpent(_wallet, _limit, _dailySpent);
+    }
```

Comment:
Now that we are routing all the writes to the different storages in the `VersionManager` I think we should add a generic `invokeStorage` method that will check that the target is legit. Otherwise we cannot add new storages in the future without doing a (costly) `VersionManager` upgrade in every wallet.

Something like what you had before:
```
function invokeStorage(address _wallet, address _storage, bytes _data) external onlyFeature(_wallet) {
   require(isStorage[_storage], "VM: invalid storage invoked");
   _storage.call(_data);
}
```